stepper-2.0.md

```diff
@@ -184,6 +181,33 @@ using Stepper 2.0's custom float format described below.
 8  END_SYSEX                               (0xF7)
 ```
 
+**Stepper set maxSpeed**
+
+Sets the maximum speed in steps per second
```

Comment:
We have to decide here if we want to specify in steps/second, in which case we probably need to use the same 4 byte float we're using for speed and accel. Or we can allow specifying steps/sec as an integer here since it will always be > 0 and < 1000 and we shouldn't lose too much in rounding. However we should probably call out the need to round if that is the route we want to take. I prefer the former for consistency, but the later for simplicity.